src/main/java/org/apache/fineract/ReportApi/ReportService.java

```diff
@@ -0,0 +1,64 @@
+package org.apache.fineract.ReportApi;
```

Comment:
A basic naming convention is to have package names to be in lowercase.

Reply:
Fixed